rails/etsy_product_catalog/etsy

```diff
@@ -0,0 +1 @@
+Subproject commit ef77c18bbddcb5adcf9f4effec2a4ddadba8fb96
```

Comment:
This concerns me. It looks like you might have set up a git repo inside a git repo. I don't think there's a file or folder called rails/etsy_product_catalog/etsy.